requirements.txt

```diff
@@ -1,4 +1,3 @@
-django>=1.11,<2.0
```

Comment:
I don't think is a good idea to remove this, use instead `django>=1.11,<2.1`
the travis build should not be affected because we'll be installing django before this requirement file is processed and the django requirement will be already satisfied.